packetbeat/protos/thrift/thrift.go

```diff
@@ -157,6 +158,11 @@ type Thrift struct {
 	Idl          *ThriftIdl
 }
 
+var (
+	unmatchedRequests  = expvar.NewInt("thriftUnmatchedRequests")
```

Comment:
Perhaps we can generalise this in the future so the methods return an error and the "stats" gathering happen one level up.